tests/CustomPost.php

```diff
@@ -8,4 +8,6 @@
 class CustomPost extends Model
 {
     protected $uuidField = 'customField';
```

Comment:
It was more for those that didn't want to call their field `uuid` for whatever reason, but I think now you mention it I'm going to remove it. More trouble than what it's worth I think 🤔